ERCS/erc-4337-agg.md

```diff
@@ -0,0 +1,145 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7766
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.